inst/shiny-examples/example3/customPlots.R

```diff
@@ -149,7 +147,7 @@ volcanoPlot <- function(df, title, subtitle) {
         subtitleScaleFontFactor = 0.5,
         showAnimation           = FALSE,
         width                   = "100%",
-        events                  = JS("{'select': function(o, e, t){if (o.y.vars.length != 0) { var g = $('#genes'); g[0].selectize.clear(); g[0].selectize.addItems(o.y.vars);} }}")
+        events                  = JS("{'select': function(o, e, t){if (o != null) { if (o.y.vars.length != 0) { var g = $('#genes'); g[0].selectize.clear(); g[0].selectize.addItems(o.y.vars);}}; }}")
```

Comment:
Check all the pieces of the object tree or you will eventually run into the same problem (for both of these you need three checks: o != null && o.y != null && o.y.vars != null)  Logical operator && will stop at the first False so you can safely check from left to right in the statement down into the tree.

Reply:
ok